posts.js

```diff
@@ -10,40 +13,57 @@ class Post {
 		this.tags = tags;
 		this.desc = desc;
 		this.time = new Date().toDateString();
-		this.id = fs.readdirSync(dir).length;
+		this.id;
```

Comment:
Maybe we can let mongo handle the id? with ``_id``

Reply:
Id is given in `save()`  to `this.id = _res;`

Reply:
ah, I see